doc/toolbox.1.md

```diff
@@ -40,6 +40,20 @@ access to the user's home directory, the Wayland and X11 sockets, networking
 (including Avahi), removable devices (like USB sticks), systemd journal, SSH
 agent, D-Bus, ulimits, /dev and the udev database, etc..
 
+## Supported distribution images
```

Comment:
We need to ensure that this section fits with the introduction above.

The introduction ends by mentioning the fedora-toolbox image, but it doesn't say anything about how the image relates to the host, if the host is used to select the image, or what happens when the host is not known.  So we should cover those points here.  Otherwise it feels a bit abrupt.

*Historically, we have been pretty brief or hand wavy about these things, because Fedora was the only supported distribution.  Now that we have added RHEL, and we are trying to spell things out more clearly, it's a good opportunity to make the documentation a bit more complete.*

So, it will be good to also include hosts here:
```
## Supported operating system distributions
```